backend/geonature/core/imports/routes/__init__.py

```diff
@@ -12,3 +14,15 @@ def list_destinations():
     destinations = Destination.query.all()
     # FIXME: filter with C permissions?
     return schema.dump(destinations, many=True)
+
+
+@blueprint.route("/destination/<destinationCode>", methods=["GET"])
+@login_required
+def get_destination(destinationCode):
+    schema = DestinationSchema(only=["module"])
+    destination = db.session.execute(
+        db.select(Destination)
+        .options(joinedload("module"))
+        .where(Destination.code == destinationCode)
```

Comment:
**Suggestion:** It seems like you are using raw SQL queries with SQLAlchemy. While this is not wrong, it is recommended to use the ORM capabilities of SQLAlchemy for better security, readability, and maintainability. You can replace the raw SQL query with SQLAlchemy ORM query. [best practice]
```suggestion
    destination = Destination.query.options(joinedload("module")).filter_by(code=destinationCode).first()
```